packages/dds/sequence/src/test/fuzz/fuzzUtils.ts

```diff
@@ -339,9 +346,18 @@ export function createSharedStringGeneratorOperations(
 	}
 
 	async function obliterateRange(state: ClientOpState): Promise<ObliterateRange> {
+		const max = state.client.channel.getLength() - 1;
+		const num1 = state.random.integer(0, max);
+		const num2 = state.random.integer(0, max);
+		const start = Math.min(num1, num2);
+		const end = Math.max(num1, num2);
+		const startSide =
+			start === end ? Side.Before : state.random.pick([Side.Before, Side.After]);
```

Comment:
I don't know if the `hasNonzeroLength` generator is enough to prevent zero-length obliterates from happening - even if start !== end, depending on the sides(for example start={pos: 1, Side.After}, end={pos:2, Side.Before}), we could still end up with a zero length obliterate range. I think we need some additional checks when assigning the side, at least until zero length is figured out.

Reply:
Yeah you're right that we generate 0 length obliterates. Fuzz tests don't catch any errors with them right now though, so I've intentionally left them in :)